sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -64,7 +64,9 @@
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "cross-env TEST_MODE=playback npm run integration-test:browser",
     "unit-test:node": "cross-env TEST_MODE=playback npm run integration-test:node",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "coverage-integration-test:node": "nyc npm run integration-test:node",
```

Comment:
Better to have this `coverage-integration-test:node` step in the CI than `integration-test:node`.

Reply:
I spent about two days trying to fix this problem via configuration files and I couldn't do it. I could have missed something. If you find time, try to fix this via Nyc, Istanbul or Mocha configurations, and let me know if you find a way. I would love to go through that approach if possible.

Reply:
I know, configs are hard, we have issues logged.